.travis.yml

```diff
@@ -1,8 +1,18 @@
-language: node_js
 sudo: true
+
+language: node_js
+
 node_js:
-  - "0.10"
-  - "0.12"
-  - "iojs"
-  - "4"
-  - "5"
+  - 6
+  - 5
+  - 4
+  - 0.12
+  - 0.10
+  - iojs
+
+cache:
+  directories:
+    - node_modules
```

Comment:
this is actually not optimal, because anything in the dep tree that has a semver-included update released will then mean that "what's tested" is not the same as "what's installed on a clean install".

node_modules should only ever be cached on travis for shrinkwrapped and/or top-level apps.